fanficfare/adapters/adapter_asianfanficscom.py

```diff
@@ -253,7 +263,6 @@ def getChapterText(self, url):
             jsonlink = soup.find('link',href=re.compile(r'/api/chapters/[0-9]+/chapter_[0-9a-z]+.json'))
             chap_json = json.loads(self._fetchUrl(jsonlink['href']))
             content = self.make_soup(chap_json['post']).find('body') # BS4 adds <html><body> if not present.
-            content.name='div' # change body to a div.
```

Comment:
What's the reason for removing this line?

Reply:
Oh my bad, I was just checking something and forgot to put that line back in.